src/main/java/de/tum/in/www1/artemis/service/quiz/QuizService.java

```diff
@@ -109,7 +90,7 @@ private void fixReferenceMultipleChoice(MultipleChoiceQuestion multipleChoiceQue
      *
      * @param dragAndDropQuestion the DragAndDropQuestion which references are to be fixed
      */
-    private void fixReferenceDragAndDrop(DragAndDropQuestion dragAndDropQuestion) {
+    public void fixReferenceDragAndDrop(DragAndDropQuestion dragAndDropQuestion) {
```

Comment:
As Timor pointed out, private methods should not be made public just for testing purposes in the test code. Consider testing them indirectly through the public method that calls them